### ISSUE TITLE
Run build.sh failed

### ISSUE THREAD

**Issue by littleeleventhwolf** [StarRocks/starrocks#3711]
OS: Ubuntu 20.04
```
-- Found AWS SDK for C++, Version: 1.9.179, Install Root:/home/universe/starrocks/thirdparty/installed, Platform Prefix:, Platform Dependent Libraries: pthread;crypto;ssl;z;curl
-- Components specified for AWSSDK: core;s3;s3-crt;transfer, application will be depending on libs: aws-cpp-sdk-s3-crt;aws-cpp-sdk-transfer;aws-cpp-sdk-s3;aws-cpp-sdk-core;aws-crt-cpp;aws-c-auth;aws-c-cal;aws-c-common;aws-c-compression;aws-c-event-stream;aws-c-http;aws-c-io;aws-c-mqtt;aws-c-s3;aws-checksums;pthread;crypto;ssl;z;curl
-- Try finding aws-cpp-sdk-core
CMake Error at /usr/local/share/cmake-3.21/Modules/CMakeFindDependencyMacro.cmake:47 (find_package):
  By not providing "Findaws-c-io.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "aws-c-io",
  but CMake did not find one.

  Could not find a package configuration file provided by "aws-c-io" with any
  of the following names:

    aws-c-ioConfig.cmake
    aws-c-io-config.cmake

  Add the installation prefix of "aws-c-io" to CMAKE_PREFIX_PATH or set
  "aws-c-io_DIR" to a directory containing one of the above files.  If
  "aws-c-io" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  /home/universe/starrocks/thirdparty/installed/lib/aws-c-http/cmake/aws-c-http-config.cmake:3 (find_dependency)
  /usr/local/share/cmake-3.21/Modules/CMakeFindDependencyMacro.cmake:47 (find_package)
  /home/universe/starrocks/thirdparty/installed/lib/aws-crt-cpp/cmake/aws-crt-cpp-config.cmake:3 (find_dependency)
  /usr/local/share/cmake-3.21/Modules/CMakeFindDependencyMacro.cmake:47 (find_package)
  /home/universe/starrocks/thirdparty/installed/lib/cmake/aws-cpp-sdk-core/aws-cpp-sdk-core-config.cmake:13 (find_dependency)
  /home/universe/starrocks/thirdparty/installed/lib/cmake/AWSSDK/AWSSDKConfig.cmake:307 (find_package)
  CMakeLists.txt:204 (find_package)
```

**Comment by imay** [comment]
Do you consider compiling StarRocks with dev-env docker image?

**Comment by littleeleventhwolf** [comment]
Fine, I'll try to compile in docker.

**Comment by kevincai** [comment]
should be fixed in PR #8226
